wallet/wallet-cli-commands/src/lib.rs

```diff
@@ -194,7 +194,11 @@ pub enum ColdWalletCommand {
     },
 
     #[clap(name = "address-show")]
-    ShowReceiveAddresses,
+    ShowAddresses {
+        /// Include the change addresses along with the receiving addresses
+        #[arg(long = "include-change")]
+        include_change: bool,
```

Comment:
looks like it should an optional argument

Reply:
it defaults to false if not specified so works as if optional